src/uwtools/resources/schema_with_metatasks.rng

```diff
@@ -0,0 +1,622 @@
+<grammar xmlns="http://relaxng.org/ns/structure/1.0" datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
```

Comment:
We likely want to add a header comment here that says we copied this file from the Rocoto repository at release version 1.3.5.